client/ayon_core/hosts/houdini/startup/MainMenuCommon.xml

```diff
@@ -104,6 +104,34 @@ parent = hou.qt.mainWindow()
 host_tools.show_experimental_tools_dialog(parent)
 ]]></scriptCode>
             </scriptItem>
+
+        <separatorItem/>
+
+        <scriptItem id="ayon_server">
+            <label>Open AYON server in browser</label>
+            <scriptCode><![CDATA[import os; __import__("webbrowser").open(os.getenv("AYON_SERVER_URL"))]]></scriptCode>
+        </scriptItem>
+
+        <subMenu id="ayon_help">
+            <label>Help...</label>
+
+            <scriptItem id="ayon_artist_docs">
+                <label>Houdini Artist Docs</label>
+                <scriptCode><![CDATA[__import__("webbrowser").open("https://ayon.ynput.io/docs/addon_houdini_artist")]]></scriptCode>
+            </scriptItem>
+
+            <scriptItem id="ayon_admin_docs">
+                <label>Houdini Admin Docs</label>
+                <scriptCode><![CDATA[__import__("webbrowser").open("https://ayon.ynput.io/docs/addon_houdini_admin")]]></scriptCode>
+            </scriptItem>
+
+            <scriptItem id="community_forums">
+                <label>Community Forums</label>
```

Comment:
Should we label this `AYON Community Forum` instead?
Just so it's clear it's AYON related - not related to whatever studio might have rolled out AYON, etc.

```suggestion
                <label>AYON Community Forum</label>
```